Firmware/tmc2130.cpp

```diff
@@ -191,10 +188,22 @@ void tmc2130_init()
 	}
 	for (uint_least8_t axis = 3; axis < 4; axis++) // E axis
 	{
-		tmc2130_setup_chopper(axis, tmc2130_mres[axis], tmc2130_current_h[axis], tmc2130_current_r[axis]);
+		tmc2130_setup_chopper(axis,
+            tmc2130_mres[axis], // this is not changed with ECool
+            tmc2130_current_h[axis] + params.enableECool,
+            tmc2130_current_r[axis] + params.enableECool);
```

Comment:
Whiskey Tango Foxtrot?
Did you meant tmc2130_current_h[axis + params.enableECool] ?

Reply:
OMG, yes I did